engines/tantivy-0.20/Cargo.toml

```diff
@@ -8,7 +8,8 @@ edition = "2021"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
-tantivy = "0.19"
+tantivy = "0.20"
+env_logger = "0.5"
```

Comment:
just curious -- is that required ?

Reply:
My bad, this is not required.... Removing now.